lib/network.sh

```diff
@@ -30,21 +30,21 @@ function prefixlen() {
 }
 
 # Provisioning Interface
-export CLUSTER_PROVISIONING_INTERFACE=${CLUSTER_PROVISIONING_INTERFACE:-"ironicendpoint"}
+export BARE_METAL_PROVISIONER_INTERFACE=${BARE_METAL_PROVISIONER_INTERFACE:-"ironicendpoint"}
```

Comment:
Fix quotes to go around the assignment instead inside the braces.

Reply:
done